org.racket_lang.Racket.yml

```diff
@@ -0,0 +1,34 @@
+app-id: org.racket_lang.Racket
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: drracket
+rename-desktop-file: drracket.desktop
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=home
+  - --filesystem=/tmp
+  - --filesystem=/var/tmp
```

Comment:
why do you need this broad filesystem access?

Reply:
for `home`, because it is fair to assume users want to use `drracket` or `racket` open `.rkt` files in their home directory. And all packages are installed under `~/.racket`

`/tmp` is needed, because racket, like other programming languages, has some apis to access the system temporary directory. 

`/var/tmp` might not be needed, though.